provisioning/roles/common/tasks/main.yml

```diff
@@ -24,12 +24,12 @@
       - xubuntu-restricted-extras
 
 - name: Create local .bin
-  shell: mkdir ~/.bin
+  file: path=~/.bin state=directory mode=0755
   become: yes
   become_user: "{{ user }}"
 
 - name: Create Desktop folder
-  shell: mkdir ~/Desktop
+  shell: path=~/Desktop state=directory mode=0755
```

Comment:
should be `file:`

Reply:
Added a new commit for this.  Good catch.